### ISSUE TITLE
relative urls in css after build

### ISSUE THREAD

**Issue by whitecolor** [stealjs/steal-tools#140]
I have the following structure

```
/assets
 /img
   /image1.png
/styles
  /some-style.less
/dist
  /bundles
   /result.css
```

```some-style.less``` contains url:
```less
background-image: url(../assets/img/image1.png)
```
after build with ```bundlesPath == 'dist/bundles'``` in built ```result.css``` there is the url:
```less
background-image: url(assets/img/image1.png)
```

I think it is not correct. It url stays relative it should be
```less
background-image: url(../../assets/img/image1.png)
```

what do you think about it?

**Comment by justinbmeyer** [comment]
@whitecolor I agree.  Is this happening still?  Can you ping me tomorrow on gitter: https://gitter.im/bitovi/steal

**Comment by whitecolor** [comment]
I think it happens. But currenlty I use only absolute paths in my less files not to mess with relative paths issues.

**Comment by matthewp** [comment]
We have `bundleAssets` now which will move assets into your `dist/` and rewrite css to point to those assets. You use it like this:

```js
stealTools.build({
  config: __dirname + "/package.json!npm"
}, {
  bundleAssets: true
});
```

**Comment by whitecolor** [comment]
@matthewp does it bundle images/fonts?

**Comment by matthewp** [comment]
Those that are in css get bundled automatically. You can also specify a `glob` other assets.

**Comment by whitecolor** [comment]
@matthewp I've looked at the docs, think they should be more elaborative )

**Comment by matthewp** [comment]
Fair enough, let's reopen this to improve the docs.

**Comment by matthewp** [comment]
Here are the bundleAssets options:  http://stealjs.com/docs/steal-tools.BundleAssetsOptions.html

Probably the place to add some examples.

**Comment by matthewp** [comment]
## Infer assets from css `url()`:

```js
stealTools.build({
  config: __dirname + "/package.json!npm"
}, {
  bundleAssets: true
});
```

## Bundle assets from a glob

In this example in addition to inferring assets from css `url()` you can also specify a glob.

```js
stealTools.build({
  config: __dirname + "/package.json!npm"
}, {
  bundleAssets: {
    glob: "images/**/*"
  }
});
```

Additional a copy of `steal.production.js` will be moved to `dist/node_modules/steal/steal.production.js`.
